app.js

```diff
@@ -60,28 +50,22 @@ const reportChannelId = config.get('slack.report_channel_id');
 const http = require('http');
 const port = process.env.PORT || 3000
 
-var isJiraOkay = true;
-var isSlackOkay = true;
-
-const server = http.createServer((req, res) => { 
+const server = http.createServer((req, res) => {
     appInsights.defaultClient.trackNodeHttpRequest({request: req, response: res});
     if (req.method !== 'GET') {
         res.statusCode = 405;
         res.end("error")
     } else if (req.url === '/health') {
-        if ((!app.receiver.client.badConnection) && isJiraOkay && isSlackOkay) {
-            res.statusCode = 200;
-        } else {
+        const connectionError = app.receiver.client.badConnection;
+        if (connectionError) {
             res.statusCode = 500;
+        } else {
+            res.statusCode = 200;
         }
-        myResponse = {
+        const myResponse = {
             status: "UP",
             slack: {
-                connection: (app.receiver.client.badConnection ? "DOWN" : "UP"),
-                status: (isSlackOkay ? "UP" : "DOWN"),
-            },
-            jira: {
-                status: (isJiraOkay ? "UP" : "DOWN"),
```

Comment:
this was just hardcoded to be up so I've removed the dead code

Reply:
It's not hardcoded. Those flags can get set to false when posting to slack or JIRA fails in the new workflow

Reply:
hmm I don't think that's the best approach.

that doesn't mean Jira is down, a single failed request shouldn't cause a healthcheck failure.

fine to alert via app insights that there's failures

Reply:
Okay, might be worth re-visiting what's reported on the health page, then